evap/locale/de/LC_MESSAGES/djangojs.po

```diff
@@ -0,0 +1,71 @@
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the PACKAGE package.
+# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
+#
+#, fuzzy
+msgid ""
+msgstr ""
+"Project-Id-Version: PACKAGE VERSION\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2023-10-16 20:14+0200\n"
+"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
+"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
+"Language-Team: LANGUAGE <LL@li.org>\n"
+"Language: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+
+#: evap/static/js/notebook.js:29 evap/static/ts/src/notebook.ts:38
+msgid "The server is not responding."
+msgstr ""
+
+#: evap/static/js/sortable_form.js:19
+msgid "Delete"
+msgstr ""
+
+#: evap/static/js/sortable_form.js:20
+msgid "add another"
+msgstr ""
+
+#: evap/static/js/tom-select.complete.min.js:132
+msgid "<div>"
+msgstr ""
```

Comment:
It seems that tomselect uses something like `_(...)` somewhere, we should probably ignore our external library files in the `makemessage` call

Reply:
I am not sure which rule would apply to ignore only the external libraries and not our own files.

Reply:
From the documentation, I think adding something like [`--ignore=evap/static/js/*.min.js`](https://docs.djangoproject.com/en/4.2/ref/django-admin/#cmdoption-makemessages-ignore) to the [`makemessages` command line](https://github.com/e-valuation/EvaP/pull/2036/files#diff-80bbdb2f9832c97a33b4cebdad86c8bc7d2e3463f4db47b2bde21067b38f6154R12) should already do the trick.